priv/repo/migrations/20170424215355_add-github-id-to-user.exs

```diff
@@ -0,0 +1,9 @@
+defmodule :"Elixir.CodeCorps.Repo.Migrations.Add-github-id-to-user" do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :github_id, :string, default: ""
```

Comment:
@npendery i looked into this and I was running into an issue with the guardian serializer serializing a null integer. I'm not sure what the issue is. I'll look into